tests/mock_tables/mock_multi_asic.py

```diff
@@ -14,6 +14,8 @@ def mock_is_multi_asic():
 def mock_get_namespace_list(namespace=None):
     return ['asic0', 'asic1']
 
+
 multi_asic.get_num_asics = mock_get_num_asics
 multi_asic.is_multi_asic = mock_is_multi_asic
 multi_asic.get_namespace_list = mock_get_namespace_list
+multi_asic.get_namespaces_from_linux = mock_get_namespace_list
```

Comment:
why is this needed? the value is the same as L20

Reply:
Multi ASIC CLI uses this API so needs to be mocked.

Reply:
I was referring to this var 'get_namespaces_from_linux'. but saw its definition in multi_asic.py. So all good

Reply:
me too :)